src/style.css

```diff
@@ -1 +1,66 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url(./fonts/Avenir-Book.ttf) format('truetype'),
+    url(./fonts/Avenir.ttc) format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
```

Comment:
Using the same `font-family` name 'Avenir' for different font weights might cause conflicts. Consider using different names or specifying the weight in the `font-family` name to differentiate them.